src/components/dropdown/README.md

```diff
@@ -0,0 +1,140 @@
+# Dropdown
+
+A standard dropdown.
+
+## Usage
+
+```jsx
+import { Dropdown } from '@nomios/web-uikit';
+
+const options = [
+    { value: 'car', label: 'Car' },
+    { value: 'bike', label: 'Bike' },
+    { value: 'truck', label: 'Truck' },
+];
+
+<Dropdown options={ options } />
+```
+
+### With custom options
+
+```jsx
+import { Dropdown } from '@nomios/web-uikit';
+
+const customOption = ({ defaultClassName, data }) => (
+    <div className={ defaultClassName }>My value is: { data.value }</div>
+);
+
+const options = [
+    { value: 'car', render: customOption },
```

Comment:
I feel that we should have a `renderOption`, similar to `renderTrigger` in the Dropdown instead. We could still allow this and it would take precedence. This suggestion is simply to avoid repetition. What do you think?

Reply:
Yep, seems fair! 👍